src/chainstate/burn/db/sortdb.rs

```diff
@@ -2823,9 +2823,11 @@ impl SortitionDB {
 
         sortition_db_handle.store_transition_ops(&new_snapshot.0.sortition_id, &new_snapshot.1)?;
 
+        announce_to(reward_set_info);
```

Comment:
There's no plausible scenario where this `announce_to()` succeeds but the subsequent commit fails, right?  Maybe the subsequent commit should `panic`, just to be sure?

Reply:
Updated to `expect` in 8608cc7fd